i18n/miraheze/en.json

```diff
@@ -92,52 +92,51 @@
 	"group-abuse-member": "{{GENDER:$1|abuse team}}",
 	"group-steward": "Stewards",
 	"group-steward-member": "{{GENDER:$1|steward}}",
-	"grouppage-steward": "m:Stewards",
+	"grouppage-steward": "m:Special:MyLanguage/Stewards",
 	"group-sysadmin": "System administrators",
 	"group-sysadmin-member": "{{GENDER:$1|system administrator}}",
-	"grouppage-sysadmin": "m:System administrators",
+	"grouppage-sysadmin": "m:Special:MyLanguage/Tech:SRE Volunteers",
 	"group-interwiki-admin": "Interwiki Administrator",
 	"group-interwiki-admin-member": "interwiki administrator",
-	"grouppage-interwiki-admin": "m:Interwiki administrators",
+	"grouppage-interwiki-admin": "m:Special:MyLanguage/Interwiki administrators",
 	"group-oversight": "Oversighters",
 	"group-oversight-member": "oversight",
-	"grouppage-oversight": "m:Oversight",
-	"group-checkuser": "Checkusers",
-	"group-checkuser-member": "checkuser",
-	"grouppage-checkuser": "m:CheckUser",
+	"grouppage-oversight": "m:OSpecial:MyLanguage/versight",
 	"group-wikicreator": "Wiki Creators",
 	"group-wikicreator-member": "{{GENDER:$1|wiki creator}}",
-	"grouppage-wikicreator": "m:Wiki creators",
+	"grouppage-wikicreator": "m:Special:MyLanguage/Meta:Wiki creators",
 	"group-wikimanager": "Wiki Managers",
 	"group-wikimanager-member": "{{GENDER:$1|wiki manager}}",
-	"grouppage-wikimanager": "m:Wiki managers",
+	"grouppage-wikimanager": "m:WSpecial:MyLanguage/iki managers",
```

Comment:
Something seems wrong here...

Also, this group doesn't even exist anymore does it?

Reply:
It doesn't exist, but let's leave it for now. I'll do another PR later to remove it. Nevertheless, I'll fix the mistake